Components/badge.css

```diff
@@ -0,0 +1,32 @@
+.img{
```

Comment:
the naming convention can be improved here. instead of adding a class for image tag as **.img**, you can add something like **.img-responsive / img-round**,  basically, the `class name should have the actual purpose` of what that class is doing